src/LemonWeb/Deployer/Database/SqlUpdate/AbstractSqlUpdate.php

```diff
@@ -29,9 +29,14 @@
      */
     final public function __construct(array $options)
     {
-        $this->options = array_merge($options, array(
-            'charset' => 'utf8',
-        ));
+        $this->options = array_merge(
+            [
+                'charset' => 'utf8',
+                'database_default_name' => 'mijnwebwinkel',
+                'database_lemoncrm_name' => 'administratie',
```

Comment:
Dat lijkt me inderdaad beter, in principe hoeven ze alleen als defaults opgenomen te worden in patches waarin ze daadwerkelijk gebruikt worden